src/renderer/meson.build

```diff
@@ -61,8 +61,7 @@ endforeach
 r_macro = ' '.join(r_macro)
 config.set('TAISEI_BUILDCONF_RENDERER_BACKENDS', r_macro)
 config.set_quoted('TAISEI_BUILDCONF_RENDERER_DEFAULT', default_renderer)
-config.set('TAISEI_BUILDCONF_HAVE_WINDOWS_ANGLE_FALLBACK',
-    host_machine.system() == 'windows' and
+config.set('TAISEI_BUILDCONF_HAVE_ANGLE',
     enabled_renderers.contains('gles30') and
```

Comment:
This check is probably not needed here, especially considering that one can enable `gles20` without `gles30`. But It may be a good idea to print a warning if `install_angle` is enabled but neither `gles20` nor `gles30` are.